modules/rgbd/src/kinfu.cpp

```diff
@@ -251,7 +251,7 @@ bool KinFuImpl<MatType>::updateT(const MatType& _depth)
         if(!success)
             return false;
 
-        pose = pose * affine;
+        pose = pose * affine.matrix;
```

Comment:
Looks like here matrix multiplication order should be opposite, `Affine3f` is quite counter-intuitive in that

Reply:
Other places should be checked where `Affine3f` multiplication was replaced by matrix multiplication.

Reply:
done

Reply:
there was only one place with this multiplication